app/controllers/admin/kudos_controller.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module Admin
+  class KudosController < AdminController
+    before_action :set_kudo, only: %i[show edit update destroy]
```

Comment:
For the sake of consistency I suggest getting rid of this callback - you are not using it in the `CompanyValueController`

Reply:
Fixed.